CONRTIBUTING.md

```diff
@@ -61,6 +61,28 @@ __The following approach aims to reconcile these two competing demands:__
 --------
 
 
+### Library Standards
+
+The team has agreed to consistently use several third-party Python packages.
+This list will likely be fairly dynamic as we are always looking for things
+that will increase our productivity. We have a strong preference for tools that
+follow the [Principle of Least Astonishment](http://c2.com/cgi/wiki?PrincipleOfLeastAstonishment)
+(also on [Wikipedia](https://en.wikipedia.org/wiki/Principle_of_least_astonishment)).
+
+The list:
+
+  - [arrow](https://pypi.python.org/pypi/arrow): Date/Time utilities
+  - [bottle](https://pypi.python.org/pypi/bottle): Web application micro-framework
+  - [cryptography](https://pypi.python.org/pypi/cryptography): cryptographic recipes and primitives
+  - [eventlet](https://pypi.python.org/pypi/eventlet): concurrent networking library
+  - [pymongo](https://pypi.python.org/pypi/pymongo): MongoDB driver
+  - [redis](https://pypi.python.org/pypi/redis): Redis client library
+  - [requests](https://pypi.python.org/pypi/requests): makes http requests virtually painless
+  - [tornado](https://pypi.python.org/pypi/tornado): concurrent networking library
```

Comment:
My answer to that question is "neither" and "both". The difficulty is that, once we include something (in this case, Tornado), we are in essence saying "you should feel free to use this and have at least some confidence that it will be here tomorrow." As I said elsewhere, omitting something is much easier than removing it later.

I'm fine leaving both eventlet and tornado in. I just want us to make that decision with our eyes wide opened: for all practical purposes this decision will be cast in stone.